### ISSUE TITLE
I get error'd on npm start after npm install :[ 

### ISSUE THREAD

**Issue by MrChristofferson** [richi1717/ellies-quest-rebirth#1]
```
 [370] ./~/strip-ansi/index.js 161 bytes {0} [built]
  [371] ./~/ansi-regex/index.js 135 bytes {0} [built]

ERROR in ./src/index.jsx
Module build failed: Error: Cannot read config package: eslint-config-defaults/configurations/eslint
Error: Cannot find module 'eslint-config-defaults/configurations/eslint'
Referenced from: C:\Users\cvandenberg\.eslintrc
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at loadPackage (C:\Users\cvandenberg\Sites\ellies-quest-rebirth\node_modules\eslint\lib\config\config-file.js:168:1
)
    at loadConfigFile (C:\Users\cvandenberg\Sites\ellies-quest-rebirth\node_modules\eslint\lib\config\config-file.js:21
:18)
    at load (C:\Users\cvandenberg\Sites\ellies-quest-rebirth\node_modules\eslint\lib\config\config-file.js:385:18)
    at C:\Users\cvandenberg\Sites\ellies-quest-rebirth\node_modules\eslint\lib\config\config-file.js:326:36
    at Array.reduceRight (native)
    at applyExtends (C:\Users\cvandenberg\Sites\ellies-quest-rebirth\node_modules\eslint\lib\config\config-file.js:309:
8)
    at Object.load (C:\Users\cvandenberg\Sites\ellies-quest-rebirth\node_modules\eslint\lib\config\config-file.js:395:2
)
    at loadConfig (C:\Users\cvandenberg\Sites\ellies-quest-rebirth\node_modules\eslint\lib\config.js:74:33)
    at getPersonalConfig (C:\Users\cvandenberg\Sites\ellies-quest-rebirth\node_modules\eslint\lib\config.js:137:22)
    at getLocalConfig (C:\Users\cvandenberg\Sites\ellies-quest-rebirth\node_modules\eslint\lib\config.js:195:53)
    at Config.getConfig (C:\Users\cvandenberg\Sites\ellies-quest-rebirth\node_modules\eslint\lib\config.js:272:22)
    at processText (C:\Users\cvandenberg\Sites\ellies-quest-rebirth\node_modules\eslint\lib\cli-engine.js:203:27)
 @ multi main
Child html-webpack-plugin for "index.html":
    chunk    {0} index.html 516 kB [rendered]
        [0] ./~/html-webpack-plugin/lib/loader.js!./build/index.html 553 bytes {0} [built]
        [1] ./~/lodash/lodash.js 516 kB {0} [built]
        [2] (webpack)/buildin/module.js 251 bytes {0} [built]
webpack: bundle is now VALID.
```

**Comment by richi1717** [comment]
Yeah I didn't actually touch this for awhile.  Sorry about that.  I just recently upgraded to React 15.5, webpack 2, etc so it's a step in the right direction.  I'm actually at a broken spot right now but am tired tonight.  Will get it working tomorrow and upgrade the version so people are aware of the breaking changes.   I dropped Immutable.js because I didn't actually need it.  If you have this project cloned already, I suggest deleting node_modules, pulling from master, and then running `yarn`.  I got rid of a ton of unnecessary modules.
